gRPC/BICgRPCServer/ClassesSource/BICListener.cpp

```diff
@@ -126,11 +126,15 @@ namespace BICGRPCHelperNamespace
     /// <param name="id"></param>
     void BICListener::onLastStimulationFunctionId(const uint16_t id)
     {
-        if (m_isStimulating)
+        try
         {
-            // Write Event Information to Console
-        std::cout << "\tSTATE INFO: Stimulation Function ID: " << id << " ." << std::endl;
+            std::cout << "\tSTATE INFO: Stimulation Function ID: " << id << " ." << std::endl;
+
```

Comment:
sorry for being nitpicky - inconsistent whitespace

Reply:
noted- adjusted whitespace